dist/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Leader board</title>
+    <script defer src="main.js"></script></head>
+    <body>
+        <main class="container">
+            <section class=".col col--leaderboard">
+                <div class="col--leaderboard__container">
+                    <h1>Leaderboard</h1>
+                    <div class="recent-scores">
+                        <h2>Recent scores</h2>
+                        <button type="button">refresh</button>
+                    </div>
+                    <ul class="col--leaderboard__list">
+                        <li class="col--leaderboard__list--item">Name: 100</li>
+                        <li class="col--leaderboard__list--item">Name: 45</li>
+                        <li class="col--leaderboard__list--item">Name: 123</li>
+                    </ul>
+                </div>
+            </section>
+            <section class=".col col--add-score">
+                <h2>Add your score</h2>
+                <form class="col--add-score__form">
+                    <label for="name" hidden>Enter your name</label>
+                    <input type="text" name="name" id="name" placeholder="Your name">
+                    <label for="score" hidden>Enter your score</label>
+                    <input type="number" name="number" id="score" placeholder="Your score" min="0">
+                    <button type="submit" class="btn-submit">Submit</button>
+                </form>
+            </section>
```

Comment:
- [ ] One of the objectives of this milestone is to practice JS ES6 modules, functions, syntaxes, and DOM. I think the plain HTML phrase is used to emphasize that styling using CSS is mandatory for this milestone. Kindly construct and render components in JS DOM which are children of the root HTML element. :pray:

Reply:
I added the list dynamically is that what you meant 😓  ?

Reply:
This issue has been addressed well. Nice job! :clap: :clap: :clap: